egs/swbd/s5c/local/chain/multi_style/run_tdnn_1a.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
```

Comment:
I think this script should go in into a new tuning directory, e.g., local/chain/multi_style/tuning/run_tdnn_1a.sh.

Then, in local/chain/multi_style, I suggest creating a symbolic link to run_tdnn_1a.sh called run_tdnn.sh. That would be analogous to what is done in other recipes, such as https://github.com/kaldi-asr/kaldi/tree/master/egs/swbd/s5c/local/chain .

Also, I suggest adding a more detailed comment at the top of this script that also includes some WER results. For example, see https://github.com/kaldi-asr/kaldi/blob/master/egs/swbd/s5c/local/chain/tuning/run_tdnn_lstm_1n.sh.

Reply:
Makes sense but instead of creating the sym link in local/chain/multi_style it is better to create in local/chain/run_tdnn_multistyle.sh, since all the other tend scripts are located there. What do you think?

Reply:
That sounds better.

Reply:
I made the changes except for the inclusion of WERs. I will include them once the tdnn finishes training. In my old experiments I never decoded on train_dev.